generator/osm2type.cpp

```diff
@@ -423,7 +423,7 @@ string MatchCity(OsmElement const * p)
       {"montreal", {-73.995802, 45.398482, -73.474295, 45.70479}},
       {"moscow", {36.9964599609, 55.3962717136, 38.1884765625, 56.1118730004}},
       {"mumbai", {72.7514648437, 18.8803004445, 72.9862976074, 19.2878132403}},
-      {"munchen", {11.3433837891, 47.9981928195, 11.7965698242, 48.2730267576}},
+      {"munchen", {11.3433837891, 47.9981928195, 11.7965698242, 48.2530267576}},
```

Comment:
Why?

Reply:
See earlier comment. I made my changes not from latest head file. Re-submitted